### ISSUE TITLE
[<ReactComponent>] args translation error

### ISSUE THREAD

**Issue by JaggerJo** [Zaid-Ajaj/Feliz#460]
The ReactComponent compiler plugin apparently can't deal with functions that take 2 arguments where the 2nd in optional. 

The code below throws the following exception.

<img width="447" alt="image" src="https://user-images.githubusercontent.com/13090415/156211311-874a9f68-a1e3-4f38-8140-c00a414277ff.png">

```fsharp

[<AutoOpen>]
module Example =

    type Person =
        { FirstName: string
          LastName: string
          Age: int }

    type UI with

        [<ReactComponent>]
        static member PersonView (person: Person, ?showAge: bool) =
            let showAge = defaultArg showAge false

            Html.div [
                Html.span person.FirstName
                Html.span person.LastName

                if showAge then
                    Html.span person.Age
            ]

        [<ReactComponent>]
        static member PersonList () =
            let people = [
                { FirstName = "Peter"
                  LastName = "Parker"
                  Age = 30 }

                { FirstName = "John"
                  LastName = "Doe"
                  Age = 40 }
            ]

            Html.div [

                for person in people do
                    UI.PersonView person
            ]
```

The first argument gets passed as the components  `props` directly. It's not wrapped in a record. 

<img width="829" alt="image" src="https://user-images.githubusercontent.com/13090415/156212984-74e3aa24-63f0-46c2-a5da-a54437a91155.png">

When the optional argument is passed everything works as expected. 

<img width="829" alt="image" src="https://user-images.githubusercontent.com/13090415/156214042-a97b6b32-258c-45de-a145-1cb88e04df7b.png">

#310 

We're on the latest version of Feliz (1.59.0)

**Comment by JaggerJo** [comment]
cc @aaronmu

**Comment by Zaid-Ajaj** [comment]
Hi @JaggerJo thanks for filing the issue and for the details. I've pushed a fix for it as of Feliz v1.60 can you please give that. a try? thank you!

**Comment by JaggerJo** [comment]
> Hi @JaggerJo thanks for filing the issue and for the details. I've pushed a fix for it as of Feliz v1.60 can you please give that. a try? thank you!

Works as expected now. Thanks a lot!
